### PR TITLE
feat: button component

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added container buttons
- added icon button
- added outline buttons

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+.outline-secondary,
+.outline-link,
+.outline-hover,
+.outline-disabled,
+.btn-icon {
+    padding: var(--s-size);
+    margin: var(--xs-size);
+    border-radius: 0.5rem;
+    font-size: var(--s-size);
+}
```
Comment (JantuDeb) [inline components/button.css]: Use a `.btn` class for common properties.

File: components/button.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    border: none;
+}
+
+.outline-primary,
+.outline-secondary,
+.outline-hover,
+.outline-link,
+.outline-disabled {
+    background-color: var(--lynx-white-color);
+}
```
Comment (JantuDeb) [inline components/button.css]: Same as above use a `.outline` class

**Review by JantuDeb — CHANGES_REQUESTED** [review]
Suggested some changes.

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+.btn-secondary,
+.btn-hover,
+.btn-link,
+.btn-disabled,
+.outline-primary,
+.outline-secondary,
+.outline-link,
+.outline-hover,
+.outline-disabled,
+.btn-icon {
```
Comment (t007rushi) [inline components/button.css]: make a utility class for this

File: components/button.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    margin: var(--xs-size);
+    border-radius: 0.5rem;
+    font-size: var(--s-size);
+}
+
+.btn-primary,
+.btn-secondary,
+.btn-hover,
+.btn-link,
+.btn-disabled {
```
Comment (t007rushi) [inline components/button.css:15]: same as above

File: components/button.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    color: var(--lynx-white-color);
+    border: none;
+}
+
+.outline-primary,
+.outline-secondary,
+.outline-hover,
+.outline-link,
+.outline-disabled {
```
Comment (t007rushi) [inline components/button.css]: same as above

**Review by t007rushi — CHANGES_REQUESTED** [review]
make suggested one

File: components/button.css (hunk 4 above)
Comment (harshitadaswani) [reply components/button.css:15]: done

**Review by JantuDeb — APPROVED** [review]
Now it's fine as you have made all the required changes, you can merge this PR.

**Review by t007rushi — APPROVED** [review]

**Pull request merged**